mantidproject/mantid
# Reflectometry GUI: Add the ability to 'Fill Down' cells

**Description of work.**
I added a fill down button which functions like excel: https://support.office.com/en-us/article/fill-a-formula-down-into-adjacent-cells-041edfe2-05bc-40e6-b933-ef48c3f308c6 Except you do not have to only add them to adjacent cells and extra rows can be selected.



**To test:**
- Open MantidPlot and Workbench
- Open ISIS Reflectometry
- Create a group and multiple rows (atleast 5)
- Select the first row and add text to one of the columns
- Select the rows you wish to 
![image](https://user-images.githubusercontent.com/24723609/60336663-ea825700-9998-11e9-87af-2ef9fcc566de.png)
- Make sure you select the column you want to "FillDown"
- Click the FillDown button
- The text you entered should now have been put into each of the rows that were selected.



refs #26025 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
